massgov/mayflower-patternlab
# DP-8440 quote organism spacing

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description
Adds component spacing to the `pullquote` organism to avoid lack of spacing between pullquotes & rich text areas

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8440)

## Steps to Test


1. View the pullquote organism in use on the `/?p=pages-information-details` page and confirm that spacing looks sufficient between the pullquote and adjacent text content. http://www.isaacchansky.com/mayflower/?p=pages-information-details 

## Screenshots



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* `@organism/by-author/pullquote` CSS

#### @TODO


*

#### Today I learned...
